app/ancillaryFactory/src/config/codepushConfig.js

```diff
@@ -0,0 +1,29 @@
+// @flow strict
+
+import CodePush from 'react-native-code-push';
+import { Platform } from 'react-native';
+
+// TODO: Fill in with Kiwi CodePush keys
```

Comment:
These should be environment variables that should not be published on GitHub, so you can delete this TODO.
Documentation available here: https://github.com/kiwicom/mobile/blob/master/docs/common/code-push.md

Reply:
We already got this covered this file should be removed